sdk/communication/Azure.Communication.Identity/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
+## 1.1.0-beta.1 (2021-02-22-preview1)
```

Comment:
Just a quick fix here, this is the date of the release rather than the API version